trite-osprey/backend/schema.sql

```diff
@@ -0,0 +1,29 @@
+DROP DATABASE IF EXISTS beermo;
+CREATE DATABASE beermo;
+
+\c beermo;
+
+DROP TABLE IF EXISTS beermo;
+CREATE TABLE beermo (
+  id SERIAL PRIMARY KEY,
+  brewery VARCHAR,
+  beername VARCHAR NOT NULL,
+  image VARCHAR,
+  price INT NOT NULL,
+  category VARCHAR,
+  description VARCHAR,
+  abu DECIMAL
+);
+
+INSERT INTO beermo ( brewery, beername, image, price, category, description, abu)
```

Comment:
a nice convention is to keep your 'fixture data' in different file. so one file has sql to update the schema. and another file like fixture-data.sql has sql to add data to the database. you can fill the database with your fixture data using the same command --- psql database_name < fixture-data.sql